frontend/markdown/en/help.md

```diff
@@ -0,0 +1,17 @@
+# Registering new users
+
+It is not possible for administrators to register new users on this page. Users need to visit the page and log in. After doing this, they will become visible in the "Edit administrators" and "Edit groupleaders" submenus in the "ADMIN" menu.
+
+# Role descriptions
+
+## Administrator
+
+The administrator role is able to appoint users to new roles, either as a new administrator or as a group leader. The role is also able to choose what catalog is active, as well as to edit or create new catalogs. This can be done in the "Edit catalogs" submenu. Administrators are not able to view the answers of other individuals on this page, but have the opportunity to download an Excel report containing the answers. This can be done in the "Download catalogs" submenu. Additionally, administrators can edit groups in the "Edit groups" submenu.
```

Comment:
The administrator role => An administrator 
The role is also able to => This role is also able to